src/sentry/sentry_metrics/indexer/limiters/cardinality.py

```diff
@@ -49,7 +49,15 @@ def _construct_quotas(use_case_id: UseCaseID) -> Optional[Quota]:
     """
 
     if use_case_id in USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS:
-        quota_args = options.get(USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS[use_case_id])
+        custom_card_limit = options.get(USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS[use_case_id])
+
+        # check that the option is set to a properly-formatted cardinality limit
+        if len(custom_card_limit) == 3:
```

Comment:
I think this should be
```python3
if not custom_card_limit:
    ...
```

Reply:
Actually never mind. I thought options is a map

Reply:
I think the default value for the option is actually `{}` so the length would be a valid check ?

Reply:
How about just
```python
if custom_card_limit:
    ...
```
I feel like checking if length is 3 might not be very readable

Reply:
Yeah, you're right. I think that would work but I didn't see it in time (I just changed it to check for len() > 0 which I think checks for the same thing). Let me know if you want me to change it to your suggestion?